sdc_interaction/SphereList obstacles
---
bool success